.github/workflows/test.yml

```diff
@@ -35,15 +35,17 @@ jobs:
 
```

Comment:
Could you change the go version above to [1.17, 1.18]? At this time, let's test it work under both 1.17 and 1.18.

Reply:
Done